programmers/K번째수.js

```diff
@@ -0,0 +1,16 @@
+function solution(array, commands) {
+    
+    let answer = [];
+    let new_arr = [];
+    
+    for (let i = 0; i < commands.length; i++) {
+        
+            new_arr = array.slice(commands[i][0] - 1, commands[i][1]);
+            new_arr.sort(
+                        (a, b) => {return a - b;} 
+                        );
```

Comment:
```suggestion
            new_arr.sort((a, b) => a - b);
```
arrow 함수에서 하나의 식만 있을 경우 return도 생략이 가능해요~